Blizzard.IO.RabbitMQ.Builders/BaseNetqRabbitBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using EasyNetQ;
+using Blizzard.IO.RabbitMQ.Extensions;
+
+namespace Blizzard.IO.RabbitMQ.Builders
+{
+    public abstract class BaseNetqRabbitBuilder
+    {
+        private static Dictionary<string, IBus> netqRabbitConnections;
```

Comment:
Initialize the dictionary here, there's no need to implement a `ctor` on this class

Reply:
f